catalogs/catalog-lakehouse-iceberg/build.gradle.kts

```diff
@@ -138,16 +138,10 @@ tasks {
 }
 
 tasks.test {
-  val skipUTs = project.hasProperty("skipTests")
-  if (skipUTs) {
-    // Only run integration tests
-    include("**/integration/**")
-  }
-
   val skipITs = project.hasProperty("skipITs")
   if (skipITs) {
     // Exclude integration tests
-    exclude("**/integration/**")
```

Comment:
why changing this?

Reply:
make it more accurate. should be `"**/integration/test/**"`